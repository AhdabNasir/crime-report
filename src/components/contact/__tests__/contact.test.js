import React from 'react';
import renderer from 'react-test-renderer';
import Contact from '..';

describe('<Contact/>', () => {
  it('renders <Contact/> component', () => {
    const tree = renderer
      .create(<Contact text="Please email" email="test@email.com" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
