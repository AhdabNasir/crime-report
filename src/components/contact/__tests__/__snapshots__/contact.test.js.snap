// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact/> renders <Contact/> component 1`] = `
<p
  className="govuk-body"
>
  Please email
   
  <a
    className="govuk-link"
    href="mailto:test@email.com"
  >
    test@email.com
  </a>
  .
</p>
`;
